src/treesa.jl

```diff
@@ -156,82 +116,76 @@ Optimize the einsum contraction pattern using the simulated annealing on tensor
 - `sc_weight` is the relative importance factor of space complexity in the loss compared with the time complexity.
 - `rw_weight` is the relative importance factor of memory read and write in the loss compared with the time complexity.
 - `initializer` specifies how to determine the initial configuration, it can be `:greedy` or `:random`. If it is using `:greedy` method to generate the initial configuration, it also uses two extra arguments `greedy_method` and `greedy_nrepeat`.
-- `nslices` is the number of sliced legs, default is 0.
-- `fixed_slices` is a vector of sliced legs, default is `[]`.
 - `greedy_config` is the configuration for the greedy method used for initializing the tree.
 
 # References
 - [Recursive Multi-Tensor Contraction for XEB Verification of Quantum Circuits](https://arxiv.org/abs/2108.05665)
 """
-Base.@kwdef struct TreeSA{RT,IT,GM,LT} <: CodeOptimizer
+Base.@kwdef struct TreeSA{RT,IT,GM} <: CodeOptimizer
```

Comment:
Please add some description in the docstring about why the `nslices` field is removed and what is the replacement.